cli/tests/pcluster/templates/test_queues_stack.py

```diff
@@ -152,6 +154,77 @@ def test_compute_nodes_dna_json(
     assert_that(json.loads(compute_node_extra_json)).is_equal_to(expected_compute_node_extra_json)
 
 
+class NetworkCard:
+    def __init__(self, index, max_interfaces=1):
+        self._index = index
+        self._max_interfaces = max_interfaces
+
+    def network_card_index(self):
+        return self._index
+
+    def maximum_network_interfaces(self):
+        return self._max_interfaces
+
+
+@pytest.mark.parametrize(
+    "efa_enabled, instance_type, network_cards_list, expected_interfaces",
+    [
+        (
+            True,
+            "p6e-gb200.36xlarge",
```

Comment:
Our logic does not check for a specific size, but for whatever GB200 size. In this case it is better in unit test to use as instance_type value `p6e.gb200.WHATEVER`.

This approach has the following advantages:
1. (functional) it tests the actual logic, where the size is ignored
2. (documentation) it uses unit test to document our behaviour, making more explicit that we ignore the size